.github/workflows/generate-docs-for-dbt-core.yml

```diff
@@ -19,149 +19,160 @@ defaults:
     shell: bash
 
 permissions:
-  contents: read
-  pull-requests: write
+  contents: write # adds new commits
+  pull-requests: write # opens new PRs
 
 jobs:
   set_variables:
     runs-on: ubuntu-latest
     outputs:
-      branch_name: ${{steps.set.outputs.BRANCH_NAME}}
-      index_artifact_name: ${{steps.set.outputs.INDEX_ARTIFACT_NAME}}
-      changie_artifact_name: ${{steps.set.outputs.CHANGIE_ARTIFACT_NAME}}
-      pr_title: ${{steps.set.outputs.PR_TITLE}}
-      pr_body: ${{steps.set.outputs.PR_BODY}}
+      branch_name: ${{ steps.set.outputs.BRANCH_NAME }}
+      index_artifact_name: ${{ steps.set.outputs.INDEX_ARTIFACT_NAME }}
+      changie_artifact_name: ${{ steps.set.outputs.CHANGIE_ARTIFACT_NAME }}
+      pr_title: ${{ steps.set.outputs.PR_TITLE }}
+      pr_body: ${{ steps.set.outputs.PR_BODY }}
 
     steps:
       - name: Set variables
         id: set
         run: |
-          echo "::set-output name=BRANCH_NAME::dbt-docs/$(date +%Y%m%d-%H%M%S)_$GITHUB_RUN_ID"
+          echo "::set-output name=BRANCH_NAME::dbt-docs-release/$(date +%Y%m%d-%H%M%S)_$GITHUB_RUN_ID"
           echo "::set-output name=INDEX_ARTIFACT_NAME::index_file"
           echo "::set-output name=CHANGIE_ARTIFACT_NAME::changie_yamls"
           echo "::set-output name=PR_TITLE::Add most recent dbt-docs changes"
-          echo "::set-output name=PR_BODY::Auto generated changes from `generate-docs-for-dbt-core.yml` workflow`"
+          echo "::set-output name=PR_BODY::Auto generated changes from `generate-docs-for-dbt-core.yml` workflow"
 
       - name: Print variables
         id: print_variables
         run: |
-          echo "Branch name:           $BRANCH_NAME"
-          echo "Index Artifact Name:   $INDEX_ARTIFACT_NAME"
-          echo "Changie Artifact Name: $CHANGIE_ARTIFACT_NAME"
+          echo "Branch name:           ${{ steps.set.outputs.BRANCH_NAME }}"
+          echo "Index Artifact Name:   ${{ steps.set.outputs.INDEX_ARTIFACT_NAME }}"
+          echo "Changie Artifact Name: ${{ steps.set.outputs.CHANGIE_ARTIFACT_NAME }}"
+          echo "PR Title:              ${{ steps.set.outputs.PR_TITLE }}"
+          echo "PR Body:               ${{ steps.set.outputs.PR_BODY }}"
 
   build_index:
     needs: [set_variables]
-    runs-on: ubuntu-latest 
+    runs-on: ubuntu-latest
+
     steps:
+
+      # TODO: switch this out for a reusable workflow so it can be called independantly.  Can't do it now since it has to be in main.
```

Comment:
This is specific for `dbt-docs` so does it need to be reusable?

Reply:
I wanted to make it reusable then also add the `on: workflow dispatch` trigger so that someone could generate the `index.html` after making changes and could then grab the index file (it will be an artifact) and stick it into `dbt-core` for testing.  This would be a bit of a bandaid for when people are unable to get `dbt-docs` running locally.  

By making it its own workflow, it can be run without generating PRs or changing anything for testing purposes.